### DISCUSSION TITLE
application/android/

### DISCUSSION THREAD

**Discussion by giscus** [KBVE/kbve#2206] [KBVE]
# application/android/

Android is an open source operating system based off of Linux that runs within the ARM architecture; the primary use-case is within the mobile/tablet eco-system but it has been expanding into server, desktop and embedables.

https://kbve.com/application/android/

**Comment by h0lybyte** [comment]
We should include a section for android emulation for mobile development.

[Android Emulator Image](https://github.com/amrsa1/Android-Emulator-image)
